src/services/pokemon.go

```diff
@@ -0,0 +1,54 @@
+package services
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/vanduc1102/ondemand-go-bootcamp/src/models"
+	"github.com/vanduc1102/ondemand-go-bootcamp/src/utils"
+)
+
+func FindById(id int) (*models.Pokemon, error) {
+	var pokemonList, error = LoadPokemonFromCSV()
+	if error != nil {
+		return nil, error
+	}
+	for _, pokemon := range pokemonList {
+		if pokemon.Id == id {
+			return &pokemon, nil
+		}
+	}
+	return nil, fmt.Errorf("pokemon id=%d does not exist", id)
+}
+
+func LoadPokemonFromCSV() ([]models.Pokemon, error) {
```

Comment:
If this function is not called outside this package it does not require to be exported.